DashboardClient/IDashboardDataClient.hpp

```diff
@@ -213,22 +228,39 @@ namespace Umati
                 ModelOpcUa::NodeId_t m_nodeId;
             };
 
+            class EventSubscriptionHandle {
+                public:
+                    EventSubscriptionHandle(int32_t clientHandle, int32_t subscriptionId) : m_clientHandle(clientHandle), m_subscriptionId(subscriptionId){};
```

Comment:
`inline` requiered at the methods, so there are no linker errors, if this file is included twice?